src/ClientData/include/ClientData/TimerChain.h

```diff
@@ -52,20 +52,20 @@ class TimerBlock {
   [[nodiscard]] size_t size() const { return data_.size(); }
   [[nodiscard]] bool at_capacity() const { return size() == kBlockSize; }
 
-  TextBox& operator[](std::size_t idx) { return data_[idx]; }
-  const TextBox& operator[](std::size_t idx) const { return data_[idx]; }
+  orbit_client_protos::TimerInfo& operator[](std::size_t idx) { return data_[idx]; }
```

Comment:
you can probably drop this one now.

Reply:
We are using it in 2 places, so I will leave it here